webpack.config.js

```diff
@@ -29,26 +28,12 @@ const config = {
     '@opentelemetry/tracing': '@opentelemetry/tracing'
   },
   plugins: [
-      new webpack.IgnorePlugin({ resourceRegExp: /^\.\/locale$/, contextRegExp: /^electron$/}),
-      new webpack.IgnorePlugin({ resourceRegExp: /^\.\/locale$/, contextRegExp: /handlebars$/ }),
-      new fileManagerPlugin({
-        events: {
-          onEnd: {
-              copy: [
-                  {
-                      source: path.join(__dirname, 'node_modules', '@microsoft', 'vscode-azext-azureutils', 'resources', '**'),
```

Comment:
this is removed due to webpack update ? Just trying to understand.

Reply:
No, I think it's actually due to [an update](https://github.com/Azure/vscode-aks-tools/pull/150/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519) a while back where we upgraded some vscode libraries that previously needed this copying, but now apparently don't. At least I've built and run it, and everything seems to be working.

This is worth double-checking though, because I'm not totally sure why this was needed in the first place. @Tatsinnit - do you remember?

Reply:
❤️❤️🙏 @peterbom, thank you for the kind ping for this, I think we have the changes coming into this PR soon, rest looks sweet, if you share VSIX we all can test

![](https://media.giphy.com/media/hVTouq08miyVo1a21m/giphy.gif)

Reply:
Actually I tested this and it is needed (for treeview icons to appear), so I've added it back.

Reply:
Icons are back:
<img width="111" alt="image" src="https://github.com/Azure/vscode-aks-tools/assets/1817696/40ea2c81-ce52-4ac2-8021-0fa077bdf89f">